packages/polymath-issuer/src/utils/parsers/index.js

```diff
@@ -83,68 +83,89 @@ const checkColumnCount = data => {
   return errorMsg;
 };
 
+const checkCSVHeaders = data => {
```

Comment:
What's the behaviour with the isAccredited and nonAccreditedLimit fields?

Reply:
I think those fields are optional (if usdsto is attached)